os.bpmsuite.executionserver/configure.sh

```diff
@@ -34,6 +34,13 @@ mkdir -p ${KIE_DIR}/repository
 chown -R jboss:root ${KIE_DIR}
 chmod -R 777 ${KIE_DIR}
 
+#temp fix for https://issues.jboss.org/browse/RHPAM-740, before the latest PostgreSQL JDGC rpm is provided
+#only copy the JDBC if it exists
+FILE=${SOURCES_DIR}/postgresql-42.2.2.jar
```

Comment:
Sorry to be nit-picky, but can you please rename FILE to POSTGRESQL_JAR (or similar)? "FILE" just seems to generic.

Reply:
no problem, will do the rename. Don't merge yet, I'm doing local testing. After it works, I will squash the commit.